### ISSUE TITLE
v13: Navigating new login screen doesn't focus buttons

### ISSUE THREAD

**Issue by bjarnef** [umbraco/Umbraco-CMS#15078]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

13.0.0

### Bug summary

When navigating the new login screen the focus isn't visible on login button and furthermore when navigating to reset password view.

### Steps to reproduce

Navigate login screen via keyboard.

**Comment by iOvergaard** [comment]
@bjarnef Thanks for reporting this. If I'm not mistaken, this would actually be an issue with the UI Library, or what do you think?

**Comment by iOvergaard** [comment]
You are right, @bjarnef. I see it as two separate issues. I'll say we should be able to fix the focus on navigation, e.g. to the reset password page. The other thing with focusable buttons will have to be fixed in the UI library, so I'll go ahead and reference your comment in a new issue on that repo.
